src/new_index/mempool.rs

```diff
@@ -520,12 +521,24 @@ impl Mempool {
             if new_txids.is_empty() {
                 break;
             }
+
+            let all_txs_count = all_txids.len();
```

Comment:
`len()` returns a precomputed field, so it shouldn't make a difference efficiency wise if we call `len()` again or keep it in a var.

But I'm fine keeping it that way too, if you prefer for it for readability.